jquery.addtocal.js

```diff
@@ -276,7 +276,7 @@
     },
 
     _renderItem: function( ul, item) {
-      return $( "<li></li>" )
+      return $( "<li class='small'></li>" )
```

Comment:
Please remove this styling from the tag, it's not a class that is part of the plugin. These should be added by the user in their own styling. The other changes seem reasonable.